src/confd/yang/infix-routing@2024-03-06.yang

```diff
@@ -185,6 +189,11 @@ module infix-routing {
     container default-route-advertise {
       description "Distribute default route to network";
       leaf enable {
+        status obsolete;
+        description "Distribute default route";
```

Comment:
Suggest changing the old description to; "Legacy, distribute default route, replaced with 'enabled'"

... or something.